howso/details_residuals.amlg

```diff
@@ -485,77 +485,78 @@
 		;pull the appropriate robust or standard residuals
 		(assign (assoc global_feature_residuals (get !residualsMap param_path)))
 
+		;if global_feature_residuals is empty, we call react_aggregate to cache the required residuals
 		(if (= 0 (size global_feature_residuals))
 			(let
 				(assoc hp_param_path (get hyperparam_map "paramPath"))
-				;output a detailed warning why global convictions aren't computed
-				(accum (assoc
-					warnings
-						(associate (concat
-							"Cannot compute 'global_case_feature_residual_convictions_full' because Full"
-							" global residuals have not been computed for action_feature "
-							(if (size action_features)
-								(concat "'" (first hp_param_path) "'")
-								"'.targetless'"
+				(let
+					(assoc
+						action_feature
+						 	(if (size action_features)
+								(first hp_param_path)
 							)
+						use_case_weights
 							(if (= ".none" (last hp_param_path))
-								" without case weights.\n"
-								(concat " with case weights from '" (last hp_param_path) "'.\n")
-							)
-							"Please call 'react_aggregate()', with "
-							(if robust_residuals "feature_residuals_robust=true" "feature_residuals_full=True")
-							(if (!= ".none" (last hp_param_path))
-								(concat ", use_case_weights=true, weight_feature='" (last hp_param_path) "'")
-								""
+								(false)
```

Comment:
weird to have if statement that explicitly outputs a false/true.   just make the value (!= ".none" ... ) and it should evaluate to false/true as needed